bitnami/airflow/README.md

```diff
@@ -93,20 +93,23 @@ The following tables lists the configurable parameters of the Airflow chart and
 | `airflow.baseUrl`                         | URL used to access to airflow web ui                                                                 | `nil`                                                        |
 | `airflow.worker.port`                     | Airflow Worker port                                                                                  | `8793`                                                       |
 | `airflow.worker.replicas`                 | Number of Airflow Worker replicas                                                                    | `2`                                                          |
+| `airflow.worker.resources`                | CPU/Memory resource requests/limits                                                                  | Memory: `256Mi`, CPU: `250m`                                 |
```

Comment:
Use this format instead:
```
| `airflow.worker.resources.limits`    | The resources limits for the Worker containers        | `{}`                        |
| `airflow.worker.resources.requests`  | The requested resources for the Worker containers     | `{}`                        |
```
Please note I'm specifying the default value as `{}` since you're commenting the default values in the **values.yaml**